tests/test_integration/test_api/test_response/test_response_mediator.py

```diff
@@ -155,12 +155,12 @@ def setUp(self):
                 "params": 'test_params',
                 'session_id': SESSION_ID,
                 'url': BASE_URL + VERSION + "/sessions/session_id",
-                "data": {'test_data': 'test_data'},
+                "data": '{"test_data": "test_data"}',
                 "headers": DEFAULT_SESSION_HEADERS,
                 "attributes": {
                     'params': 'test_params',
                     'headers': DEFAULT_SESSION_HEADERS,
-                    'data': {'test_data': 'test_data'}
+                    'data': '{"test_data": "test_data"}'
```

Comment:
There's a lot of duplication of '{"test_data": "test_data"}', let's get that into a shared variable to make it easier to change in bulk in the future.

Reply:
Fixed by removing some of the instances of this code duplication.  In some of the tests it wasn't necessary to include the data or params so I removed that from those tests.